### PR TITLE
Fix image registry extended tests and rename them

### PR DESCRIPTION
**Pull request by legionus**
@kargakis please review

### CODE REVIEW

File: sjb/config/test_cases/test_pull_request_image_registry_e2e.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
```
Comment (kargakis) [inline sjb/config/test_cases/test_pull_request_image_registry_e2e.yml]: Can you also rename this file to `test_pull_request_registry_extended.yml`?

Comment (kargakis) [reply sjb/config/test_cases/test_pull_request_image_registry_e2e.yml]: Sorry, `test_pull_request_image_registry_extended.yml`

Comment (legionus) [reply sjb/config/test_cases/test_pull_request_image_registry_e2e.yml]: @kargakis sure. done.

**Comment by kargakis** [comment]
Duh, I forgot to ask you to `sjb/generate.sh`. Opened https://github.com/openshift/aos-cd-jobs/pull/863

**Comment by kargakis** [comment]
Also https://github.com/openshift/aos-cd-jobs/pull/865

**Pull request merged**